bootstrap/sql/migrations/native/1.4.5/mysql/schemaChanges.sql

```diff
@@ -0,0 +1,60 @@
+ALTER TABLE thread_entity
+ADD COLUMN hash_id VARCHAR(32) GENERATED ALWAYS AS (MD5(id)) STORED;
```

Comment:
instead of doing this, lets mark the id as UUID column for postgres and for mysql lets use binary(16)
all we need to at that point is to make sure to call UUID_TO_BIN(id) while inserting and while reading back BIN_TO_UUID().
Lets not create additional hash columns which is not required here

Reply:
Noted